examples/vulkan/Vulkan.cpp

```diff
@@ -931,9 +931,9 @@ class VulkanExample
                 return;
             }
 
-            std::vector<std::uint32_t> buffer(static_cast<std::size_t>(file.getSize()) / sizeof(std::uint32_t));
+            std::vector<std::uint32_t> buffer(file.getSize().value() / sizeof(std::uint32_t));
 
-            if (file.read(buffer.data(), file.getSize()) != file.getSize())
+            if (file.read(buffer.data(), file.getSize().value()) != file.getSize())
```

Comment:
Nitpick but we probably don't want to promote excessive use of `.value()`:

```suggestion
const auto fileSize = file.getSize().value();
std::vector<std::uint32_t> buffer(fileSize / sizeof(std::uint32_t));

if (file.read(buffer.data(), fileSize) != file.getSize())
```